app/controllers/applications/students_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Applications::StudentsController < ApplicationController
+  # include ApplicationsHelper
```

Comment:
There is a lot of commented code here and elsewhere. Do we still need it? If not, let's remove it. We have it in git if we decide to bring it back.